src/hooks/useDappConfig.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useQuery } from '@apollo/client';
+import { EnvironmentsEnum } from '@elrondnetwork/dapp-core/types';
+import { DAPP_CONFIG } from 'api/queries';
+
+export const useDappConfig = () => {
+  const { data } = useQuery(DAPP_CONFIG);
+
+  return React.useMemo(() => {
+    const environment = EnvironmentsEnum + '.' + data.environment;
```

Comment:
We need to convert data.environment to EnvironmentsEnum, instead of concatenating them